components/serialization/text/CHANGELOG.md

```diff
@@ -1,26 +0,0 @@
-# Changelog
-
-All notable changes to this project will be documented in this file.
-
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
removing the multiple changelogs won't work unless we agree on aligning the versions across components. Can you please revert that aspect until we reach a conclusion on #119?

Reply:
Ok, I revert this change, I felt the freedom of removing them as I noticed that most are outdated in different ways, but I agree on reverting 👍